.github/workflows/nightly.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           source activate peft
           pip install -e . --no-deps
           pip install pytest-reportlog
-      
+
```

Comment:
Why this diff is here?

Reply:
Note that I moved the test to the proper unit tests using a trick that Zach showed, so I undid all the changes here (i.e. trailing whitespaces are back).